web/cobrands/fixmystreet.com/base.scss

```diff
@@ -299,3 +299,57 @@ html.lazyload .js-lazyload {
     margin-bottom: 0;
   }
 }
+
+.postcode-form-test {
+    margin: 0 -1em;
+    padding: 1em;
+    background-color: $primary;
+
+    label {
+        margin: 0;
+    }
+
+    #geolocate_link {
+        background: transparent !important;
+        border: none;
+        height: auto !important;
+        margin-top: 0 !important;
+        padding: 1em !important;
+
+        &:before {
+            vertical-align: -0.2em;
+        }
+    }
+}
+
+// Overloaded selector to beat input[type="text"]
+input.postcode-form-test__postcode {
```

Comment:
The border of this now doesn't look like other text inputs (or itself before), is that deliberate?

Reply:
Yes, intentional. It looked a bit weird before. This brings it closer to our normal input styling, but I’ve swapped the inner shadow for a drop shadow, to make it stand out from the yellow background.

Reply:
Let the record show – it turns out I’d missed a `border:` rule, so browsers were doing inconsistent things with the input. Turns out Chrome’s idea of a default input border is a little… garish 😉